sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson

```diff
@@ -0,0 +1,1493 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "ODOT_TIMS_Road_Inventory",
+        "name": "ODOT (Ohio) Road Inventory",

```

Comment:
There’s no need to disambiguate ODOT, because editors only show this layer when you’re already looking at Ohio, not Oklahoma or Oregon.